### PR TITLE
Added completion for ngrok tool

### PR DESCRIPTION
**Pull request by tbhaxor**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] If my change requires a change to the documentation, I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] If I have added a new file, I also added it to ``clean_files.txt`` and formatted it using ``lint_clean_files.sh``.
- [ ] I have added tests to cover my changes, and all the new and existing tests pass.

### CODE REVIEW

File: completion/available/ngrok.completion.bash [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# shellcheck shell=bash
+
+if _binary_exists ngrok; then
```
Comment (NoahGorny) [inline completion/available/ngrok.completion.bash:3]: no need for this check, as the completion is not using the binary itself

**Review by NoahGorny — CHANGES_REQUESTED** [review]
@tbhaxor looks great and professionally done!
I had a small comment, address it and its LGTM :smile:

File: completion/available/ngrok.completion.bash (hunk 1 above)
Comment (tbhaxor) [reply completion/available/ngrok.completion.bash]: @NoahGorny Done

**Comment by tbhaxor** [comment]
> @tbhaxor looks great and professionally done!
> I had a small comment, address it and its LGTM

Thanks man :smile:

**Review by NoahGorny — APPROVED** [review]
Well done @tbhaxor

**Comment by NoahGorny** [comment]
@tbhaxor I added some commits, you are welcome to take a look before I merge :smile:

**Pull request merged**